core/src/main/java/hudson/Util.java

```diff
@@ -901,6 +916,19 @@ public static String encode(@NonNull String s) {
      */
     @NonNull
     public static String rawEncode(@NonNull String s) {
+        return encode(s, uriMap);
+    }
+
+    /**
+     * Encode a single path component for use in an HTTP URL.
+     * Escapes all special characters for use in JNLP shell command.
```

Comment:
This is in a util function not sure why jnlp is mentioned here? It can be used anywhere

Reply:
I was mainly trying to differentiate this from the existing function, I'll update the wording on the comment